### PR TITLE
test: calculator and webpage tests

### PR DESCRIPTION
**Pull request by wufe**

### CODE REVIEW

File: test/calculator.test.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import { expect } from "chai";
+import { Calculator, Operation } from "../src/Calculator";
+
+describe('Calculator', () => {
```
Comment (Pigotz) [inline test/calculator.test.ts]: GG here! :clap: 

Just the parenthesis are not compliant to the [Allman](https://en.wikipedia.org/wiki/Indentation_style#Allman_style) style :ok_man:

File: test/webpage.test.ts [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+import * as ServerMock from "mock-http-server";
+
+import { join } from "path";
+
+import { promisify } from "util";
+import { readFile } from "fs";
+const readFilePromise = promisify(readFile);
+
+import { Webpage } from "../src/Webpage";
```
Comment (Pigotz) [inline test/webpage.test.ts]: Please, reorder the imports and move the other statements after them.

You should follow this order of imports, also by grouping similar ones together:
1. Absolute first
2. Relative then

File: test/webpage.test.ts [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+            path: '/',
+            reply: {
+                status: 200,
+                headers: { 'Content-Type': 'application/json' },
+                body
+            }
+        });
+
+        const content = await webpage.getWebpage('http://localhost:9000/');
+        expect(content).to.eq(body);
```
Comment (Pigotz) [inline test/webpage.test.ts:47]: A space before the `expect` would improve readability :raised_hands:

File: test/webpage.test.ts [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+        await expect(webpage.getWebpage('http://localhost:9000/')).to.be.rejected;
+    });
+
+    it('should save webpage content after its retrieval', async () => {
+
+        const writeFile: sinon.SinonStub<[url: string, path: string], Promise<string>> =
+            sinon.stub(webpage, <keyof typeof webpage>'_writeFile');
+
+        const body = JSON.stringify({ status: 'OK', nonce: Math.random() });
+        server.on({
```
Comment (Pigotz) [inline test/webpage.test.ts:77]: A space before the `server.on` would improve readability :raised_hands:

File: test/webpage.test.ts [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+        const contentFilePath = '.not-persisted-content';
+        await webpage.saveWebpage('http://localhost:9000/', contentFilePath);
+        
+        writeFile.restore();
+        sinon.assert.calledWith(writeFile, contentFilePath, body);
+    });
+
+    it('should save webpage content on filesystem after its retrieval', async () => {
+        const body = JSON.stringify({ status: 'OK', nonce: Math.random() });
+        server.on({
```
Comment (Pigotz) [inline test/webpage.test.ts:98]: A space before the `server.on` would improve readability :raised_hands:

File: test/webpage.test.ts [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+
+import { join } from "path";
+
+import { promisify } from "util";
+import { readFile } from "fs";
+const readFilePromise = promisify(readFile);
+
+import { Webpage } from "../src/Webpage";
+
+describe('Webpage', () => {
```
Comment (Pigotz) [inline test/webpage.test.ts]: [Allman](https://en.wikipedia.org/wiki/Indentation_style#Allman_style) style needed :ok_man:

**Review by Pigotz — COMMENTED** [review]
As usual, the implementation is fine :raised_hands: 

Just a couple of style issues :ok_hand:

File: test/calculator.test.ts (hunk 1 above)
Comment (wufe) [reply test/calculator.test.ts]: Should be fixed

File: test/webpage.test.ts (hunk 2 above)
Comment (wufe) [reply test/webpage.test.ts]: Reordered the imports and moved other statements underneath

File: test/webpage.test.ts (hunk 3 above)
Comment (wufe) [reply test/webpage.test.ts:47]: Added new line

File: test/webpage.test.ts (hunk 4 above)
Comment (wufe) [reply test/webpage.test.ts:77]: Added new line for readability

File: test/webpage.test.ts (hunk 5 above)
Comment (wufe) [reply test/webpage.test.ts:98]: Added new line

File: test/webpage.test.ts (hunk 6 above)
Comment (wufe) [reply test/webpage.test.ts]: Fixed

**Comment by Pigotz** [comment]
Perfect! :+1:

**Pull request closed without merging**